test/config-next/va-remote-a.json

```diff
@@ -30,7 +30,8 @@
       "CAAAccountURI": true
     },
     "accountURIPrefixes": [
-      "http://boulder:4000/acme/reg/"
+      "http://boulder:4000/acme/reg/",
+      "http://localhost:4001/acme/acct/"
```

Comment:
Huh, how come this is `localhost` rather than `boulder`?

Reply:
Fixed in 58e8bfd5824d1c9aff8189b035cc843077fac035.

The v1 tests were using `boulder:4000` for the directory, while the v2 tests were using `localhost:4001`. Tweaked the v2 tests to use `boulder:4001`.